src/main/java/com/google/api/codegen/ApiaryConfig.java

```diff
@@ -74,6 +74,13 @@
       HashBasedTable.<String, String, String>create();
 
   /**
+   * Specifies the pattern of the field. The pattern is expressed as a regular expression, like
```

Comment:
nit: both here and in the previous comment, I suggest s/of the field/of each field/ for clarity